src/main/java/seedu/address/logic/commands/MarkDeadlineFinishedCommand.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.task.DeadlineTask;
+
+public class MarkDeadlineFinishedCommand extends Command {
+
+    public static final String COMMAND_WORD = "fin-deadline";
```

Comment:
Probably should be `fin-due` to be consistent with the rest of the commands, but eh, they should be replaced in the future anyway.